endpoints/citrus-http/src/test/java/com/consol/citrus/http/integration/HttpMessageControllerJavaIT.java

```diff
@@ -56,7 +56,7 @@ public void httpMessageControllerIT() {
                     .get()
                     .message(new HttpMessage()
                         .method(HttpMethod.GET)
-                        .contentType("text/html")
+                        .header("contentType","text/html")
```

Comment:
these lines are not properly formatted:

```suggestion
                        .header("contentType", "text/html")
```